src/utils/privileges.cpp

```diff
@@ -26,6 +27,8 @@
 
 
 namespace my_cap {
+
+std::string home_dir;
```

Comment:
Global not needed, instead just return;
We can make a struct for it like

struct t_changes_from_sudo {
  std::string m_home_dir;
};

and return that